cli/cmd/patchies/main_test.go

```
@@ -0,0 +1,83 @@
+	}
+	if output.String() != "Remote Control token: Mount path: " {
+		t.Fatalf("prompt output = %q", output.String())
+	}
+}
+
+func TestReadTokenFileDescriptor(t *testing.T) {
+	read, write, err := os.Pipe()
+	if err != nil {
+		t.Fatalf("create pipe: %v", err)
+	}
+	defer read.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `read.Close` is not checked

(errcheck)